src/main/java/uk/ac/cam/cl/dtg/util/email/MailJetApiClientWrapper.java

```diff
@@ -0,0 +1,181 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.util.email;
+
+import com.google.inject.Inject;
+import com.mailjet.client.ClientOptions;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.errors.MailjetClientRequestException;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.resource.Contact;
+import com.mailjet.client.resource.ContactManagecontactslists;
+import com.mailjet.client.resource.Contactdata;
+import com.mailjet.client.resource.Contacts;
+import com.mailjet.client.resource.ContactslistImportList;
+import org.apache.commons.lang3.Validate;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.ac.cam.cl.dtg.util.PropertiesLoader;
+
+import javax.ws.rs.core.Response;
+
+import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
+
+public class MailJetApiClientWrapper {
+
+    private static final Logger log = LoggerFactory.getLogger(MailJetApiClientWrapper.class);
+    private final MailjetClient mailjetClient;
+    private final String newsListId;
+    private final String eventsListId;
+
+    /**
+     *  Wrapper for MailjetClient class.
+     * @param properties - needed to get the MailJet keys and list IDs
+     */
+    @Inject
+    public MailJetApiClientWrapper(final PropertiesLoader properties) {
+        ClientOptions options = ClientOptions.builder()
+                .apiKey(properties.getProperty(MAILJET_API_KEY))
+                .apiSecretKey(properties.getProperty(MAILJET_API_SECRET))
+                .build();
+
+        this.mailjetClient = new MailjetClient(options);
+        this.newsListId = properties.getProperty(MAILJET_NEWS_LIST_ID);
+        this.eventsListId = properties.getProperty(MAILJET_EVENTS_LIST_ID);
+    }
+
+    /**
+     *  Get user details for an existing MailJet account
+     *
+     * @param mailjetIdOrEmail - email address or MailJet user ID
+     * @return JSONObject of the MailJet user
+     * @throws MailjetException  - if underlying MailjetClient throws an exception
+     */
+    public JSONObject getAccountByIdOrEmail(final String mailjetIdOrEmail) throws MailjetException {
+        if (null == mailjetIdOrEmail) {
+            return null;
+        }
+        MailjetRequest request = new MailjetRequest(Contact.resource, mailjetIdOrEmail);
+        MailjetResponse response = mailjetClient.get(request);
+        JSONArray responseData = response.getData();
+        if (response.getTotal() == 1) {
+            return responseData.getJSONObject(0);
```

Comment:
Would be nice if we could marshall this into an object that has types etc, although in the short term this is probably ok for early testing.